cloud/pkg/cloudhub/servers/httpserver/pre_server.go

```diff
@@ -63,6 +64,11 @@ func createCAToSecret(ctx context.Context) error {
 		// Check whether the ca exists in the secret
 		caSecret, err := client.GetSecret(ctx, CaSecretName, constants.SystemNamespace)
 		if err != nil {
+			if !apierror.IsNotFound(err) {
+				klog.Errorf("get secret: %s error: %v", CaSecretName, err)
+				return err
```

Comment:
```go
return fmt.Errorf("get secret: %s error: %v", CaSecretName, err)
```
This makes it easier for us to trace errors. The log have no id to associated with the error stack.